.gitignore

```diff
@@ -73,6 +73,7 @@ docs/_build/
 *.crt
 *.csr
 *.key
+dojo/db_migrations/*
```

Comment:
Why was this folder added as ignore?

Reply:
We don't generate the migrations at build time?